easybuild/easyconfigs/s/Sailfish/Sailfish-0.7.4-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'Sailfish'
+version = '0.7.4'
+
+homepage = 'http://www.cs.cmu.edu/~ckingsf/software/sailfish/'
+description = """Sailfish is a software tool that implements a novel, alignment-free algorithm for the estimation of
+ isoform abundances directly from a set of reference sequences and RNA-seq reads. """
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/kingsfordgroup/sailfish/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = [
+    'Sailfish-%(version)s_bypass-C++11-check.patch',
+    'Sailfish-%(version)s_skip-included-Jellyfish.patch',
+]
+
+python = 'Python'
+pyver = '2.7.10'
+versionsuffix = '-%s-%s' % (python, pyver)
+
+builddependencies = [
+    ('CMake', '3.3.2'),
+]
+
+dependencies = [
+    ('Boost', '1.59.0', versionsuffix),
+    ('tbb', '4.3.6.211', '', True),
+    ('Jellyfish', '2.2.3'),
+    ('g2log', '1.0'),
+]
+
+preconfigopts = "export CMAKE_INCLUDE_PATH=$CMAKE_INCLUDE_PATH:$EBROOTTBB/tbb/include &&"
```

Comment:
only to `$CPATH`, but should probably be sufficient, indeed...

will check whether this is really needed